ddtrace/contrib/aiohttp/patch.py

```diff
@@ -73,6 +87,9 @@ async def _traced_clientsession_request(aiohttp, pin, func, instance, args, kwar
         # set component tag equal to name of integration
         span.set_tag_str("component", config.aiohttp_client.integration_name)
 
+        # set span.kind tag equal to type of request
+        span.set_tag_str(SPAN_KIND, SpanKind.SERVER)
```

Comment:
```suggestion
        span.set_tag_str(SPAN_KIND, SpanKind.CLIENT)
```

Reply:
everything in this file is for the client, and everything in `middlewares` is for the server (as far as I can tell)

Reply:
Yup, good catch, fixed